src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Square extends Figure {
+    public static final int MAXIMUM_SIZE = 50;
+    private int side;
+    private Random random = new Random();
```

Comment:
The 'Random' object should be created in the 'FigureSupplier' class, not inside the figure classes. Remove the 'Random' instance from this class and pass the value for 'side' through the constructor.